### PR TITLE
libavrocpp: add version 1.11.3, compatibility conan v2

### PR DESCRIPTION
**Pull request by glywk**
### Summary
Changes to recipe:  **libavrocpp/1.11.3**

#### Motivation
Add release 1.11.3, bug fix, Conan v2

#### Details
- Add release libavroccp/1.11.3
- Fix conan v2 compatibility
- Fix minimum c++14 requirement with some components of boost > 1.81.0

---
- [ x ] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [ x ] Checked that this PR is not a duplicate: #19911 , #22320 (https://github.com/conan-io/conan-center-index/discussions/24240)
- [ x ] Tested locally with at least one configuration using a recent version of Conan

### CODE REVIEW

**Comment by glywk** [comment]
Hi, how can I launch again Conan v1/v2 validation pipelines?

File: recipes/libavrocpp/all/conanfile.py
Comment (RubenRBS) [inline recipes/libavrocpp/all/conanfile.py:52]: Unfortunately we'll need to keep these requirements pinned to a specific version -while we have added some libraries to the list of allowed recipes, boost/snappy are not part of those for now :/

**Review by RubenRBS — COMMENTED** [review]
Some comments, otherwise looks good :)

**Comment by glywk** [comment]
Hi,
As requested, I have replace versions range by only one. Does it miss something else?
Regards

**Review by RubenRBS — APPROVED** [review]

**Comment by glywk** [comment]
Hi, how to get another reviewer to allow to merge this pull request?
Regards

**Comment by glywk** [comment]
Hi, I have been added to authorized user but I cannot approve this pull request. Can you help me?
Regards

**Comment by AbrilRBS** [comment]
Hi @glywk, this PR is just waiting for someone from the team to have the time to give an extra review, there's nothing extra you need to do :) Thanks a lot for your patience while we process it!

**Review by czoido — APPROVED** [review]

**Comment by glywk** [comment]
> Hi @glywk, this PR is just waiting for someone from the team to have the time to give an extra review, there's nothing extra you need to do :) Thanks a lot for your patience while we process it!

Thank you for your explanation and your validation.
Regards

**Pull request merged**
